api/cloudprotocol/cloudprotocol.go

```diff
@@ -192,33 +197,24 @@ type QueueInfo struct {
 
 // InstanceFilter instance filter structure.
 type InstanceFilter struct {
-	ServiceID string  `json:"serviceId"`
+	ServiceID string  `json:"serviceId,omitempty"`
 	SubjectID *string `json:"subjectId,omitempty"`
 	Instance  *uint64 `json:"instance,omitempty"`
 }
 
-// RequestServiceCrashLog request service crash log message.
-type RequestServiceCrashLog struct {
+// LogFilter request log message.
+type LogFilter struct {
+	From    *time.Time `json:"from"`
+	Till    *time.Time `json:"till"`
+	NodeIDs *[]string  `json:"nodeIds"`
 	InstanceFilter
-	LogID string     `json:"logId"`
-	From  *time.Time `json:"from"`
-	Till  *time.Time `json:"till"`
 }
 
-// RequestServiceLog request service log message.
-type RequestServiceLog struct {
-	InstanceFilter
-	LogID string     `json:"logId"`
-	From  *time.Time `json:"from"`
-	Till  *time.Time `json:"till"`
-}
-
-// RequestSystemLog request system log message.
-type RequestSystemLog struct {
-	LogID  string     `json:"logId"`
-	NodeID string     `json:"nodeId,omitempty"`
-	From   *time.Time `json:"from"`
-	Till   *time.Time `json:"till"`
+// RequestLog request log message.
+type RequestLog struct {
+	LogID   string    `json:"logId"`
+	LogType string    `json:"logType"`
+	Filters LogFilter `json:"filters"`
```

Comment:
`filter`?

Reply:
according documentation 
https://kb.epam.com/display/EPMDAEPRA/Requesting+logs+from+Units
it's should be `Filters`. 
Should we change to `Filter` ?

Reply:
yes

Reply:
Changed to `Filter`